scripts/ckan_migration/migrate_harvest_sources.py

```diff
@@ -262,11 +325,18 @@ def migrate_source(api_token, limit=0, debug=False, harvester_url=None):
     if not CONFIG.harvester_url.endswith("/"):
```

Comment:
should we add the same default override for this as you have for `CONFIG.new_catalog_url`?

Reply:
It's there, it just off the top of this diff view